config/default.json

```diff
@@ -28,7 +28,6 @@
     "idamApi": "https://idam-api.platform.hmcts.net",
     "idamWeb": "https://hmcts-access.service.gov.uk",
     "rdProfessionalApi": "http://rd-professional-api-prod.service.core-compute-prod.internal",
-    "s2s": "http://rpe-service-auth-provider-prod.service.core-compute-prod.internal",
```

Comment:
Query - why have you removed the s2s config setting?

Reply:
Put it back in